src/components/EditRecipe/EditRecipe.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import * as dispatchers from "../../actions/actionCreators"
+import Step1 from "./EditRecipeForm/Step1"
+import Step2 from "./EditRecipeForm/Step2";
+import Step3 from "./EditRecipeForm/Step3"
+import Step4 from "./EditRecipeForm/Step4";
+import Footer from "../navigation/footerNav/FooterNav"
+
+
+const EditRecipe = ({match}) => { 
+
+  const block =  match.params.block ? match.params.block.trim() : '';
+
+  const [page, setPage] = useState(1);
+
+  const goForward = e => {
+    e.preventDefault();
+    setPage(page + 1);
+
+  };
+  switch(block){
+    case 'desc':
+    return (
+      <div className="App">
+      <Step1 goFoward={null}/>
```

Comment:
What's up with `goForward={null}`? :/

Reply:
yh!!! it's to give the option of setting `goFoward` to `{goFoward}` when the case is anything but `desc`. This will be implemented when we create a function for **edit entire recipe** or `/editrecipe` route.